bundles/org.openhab.binding.millheat/src/main/java/org/openhab/binding/millheat/internal/handler/MillheatAccountHandler.java

```diff
@@ -0,0 +1,449 @@
+import org.eclipse.smarthome.core.library.types.QuantityType;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.ThingUID;
+import org.eclipse.smarthome.core.thing.binding.BaseBridgeHandler;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.openhab.binding.millheat.internal.MillheatCommunicationException;
+import org.openhab.binding.millheat.internal.client.BooleanSerializer;
+import org.openhab.binding.millheat.internal.client.RequestLogger;
+import org.openhab.binding.millheat.internal.config.MillheatAccountConfiguration;
+import org.openhab.binding.millheat.internal.discovery.MillheatDiscoveryService;
+import org.openhab.binding.millheat.internal.dto.AbstractRequest;
+import org.openhab.binding.millheat.internal.dto.AbstractResponse;
+import org.openhab.binding.millheat.internal.dto.DeviceDTO;
+import org.openhab.binding.millheat.internal.dto.GetHomesRequest;
+import org.openhab.binding.millheat.internal.dto.GetHomesResponse;
+import org.openhab.binding.millheat.internal.dto.GetIndependentDevicesByHomeRequest;
+import org.openhab.binding.millheat.internal.dto.GetIndependentDevicesByHomeResponse;
+import org.openhab.binding.millheat.internal.dto.HomeDTO;
+import org.openhab.binding.millheat.internal.dto.LoginRequest;
+import org.openhab.binding.millheat.internal.dto.LoginResponse;
+import org.openhab.binding.millheat.internal.dto.RoomDTO;
+import org.openhab.binding.millheat.internal.dto.SelectDeviceByRoomRequest;
+import org.openhab.binding.millheat.internal.dto.SelectDeviceByRoomResponse;
+import org.openhab.binding.millheat.internal.dto.SelectRoomByHomeRequest;
+import org.openhab.binding.millheat.internal.dto.SelectRoomByHomeResponse;
+import org.openhab.binding.millheat.internal.dto.SetDeviceTempRequest;
+import org.openhab.binding.millheat.internal.dto.SetRoomTempRequest;
+import org.openhab.binding.millheat.internal.dto.SetRoomTempResponse;
+import org.openhab.binding.millheat.internal.model.Heater;
+import org.openhab.binding.millheat.internal.model.Home;
+import org.openhab.binding.millheat.internal.model.MillheatModel;
+import org.openhab.binding.millheat.internal.model.ModeType;
+import org.openhab.binding.millheat.internal.model.Room;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.ServiceRegistration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+/**
+ * The {@link MillheatAccountHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Arne Seime - Initial contribution
+ */
+public class MillheatAccountHandler extends BaseBridgeHandler {
```

Comment:
Add `@NonNullByDefault`?
```suggestion
@NonNullByDefault
public class MillheatAccountHandler extends BaseBridgeHandler {
```

Reply:
Done